config/samples/varnishcluster.yaml

```diff
@@ -6,12 +6,12 @@ spec:
   # The number of varnish instances to run. Default: 1
   replicas: 1
   # updateStrategy can be used to control the way Varnish pods will be updated. "OnDelete" is by default.
-  updateStrategy:
-    type: "OnDelete" #can be "OnDelete", "RollingUpdate" and "DelayedRollingUpdate"
+#  updateStrategy:
+#    type: "OnDelete" #can be "OnDelete", "RollingUpdate" and "DelayedRollingUpdate"
   varnish:
     # path to image + tag
-    image: ibmcom/varnish:0.27.2
-    imagePullPolicy: Always
+#    image: ibmcom/varnish:0.27.2
```

Comment:
Commented optional fields since that sample is used to generate the ClusterServiceVersion manifest and I'd like to keep that example simple.